### ISSUE TITLE
Code coverage not measured when building from project root

### ISSUE THREAD

**Issue by ViktorHofer** [dotnet/corefx#30645]
`C:\git\corefx\src\System.Text.RegularExpressions>msbuild /t:RebuildAndTest /v:m /p:Coverage=true`. Tests pass and code coverage reports get generated but without any measurement (0% coverage)

![image](https://user-images.githubusercontent.com/7412651/41857571-2346c710-7898-11e8-9107-1c5723300c01.png)

Related: https://github.com/dotnet/corefx/issues/15850

cc @weshaggard

**Comment by weshaggard** [comment]
@pjanotti was this working for you when you doing testing on code coverage?

**Comment by pjanotti** [comment]
@weshaggard I didn't try this from the project root. @ViktorHofer try it from the tests folder, it should work just fine. I will look at this during this week.

**Comment by ViktorHofer** [comment]
Yes, it works from the tests folder of course :) Just wanted to report the unexpected behavior.

**Comment by ViktorHofer** [comment]
I don't know… But probably not?
